ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -486,8 +489,10 @@ public void shouldFillExplainQueryByIDWithCorrectInfo() {
   public void shouldReturn5xxOnSystemError() {
     // Given:
     givenMockEngine(mockEngine ->
-        EasyMock.expect(mockEngine.parseStatements(EasyMock.anyString()))
-            .andThrow(new RuntimeException("internal error")));
+        {
```

Comment:
nit: back out this change as the only change is adding unnecessary braces.

Reply:
removed the braces.